packages/airnode-deployer/terraform/aws/templates/oevGw.yaml.tpl

```diff
@@ -69,9 +71,10 @@ components:
                     type: string
 
     EndpointResponse:
-      type: object
-      additionalProperties:
-        type: string
+      type: array
+      minItems: 1
+      items:
+        type: integer
```

Comment:
This should be `string` shouldn't it? It's an array of signatures.

Reply:
Thanks.